client/views/add/add.js

```diff
@@ -59,7 +74,16 @@ Template.add.events({
   },
   // When the submit button is clicked...
   'click #modsdonebutton': function (event, template) {
-    // Checks whether the proper password was submitted
+    const modBoxes = document.getElementsByClassName('modbox');
+    if (modBoxes.length > 4) {
+      if (modBoxes.length !== 5 || modBoxes[4].value !== "") {
```

Comment:
It's better to express the logic in a single condition than nest conditionals, i.e.
```javascript
if (modBoxes.length > 4 && (modBoxes.length !== 5 || modBoxes[4].value !== "")) {
```
But this is also kind of confusing. Right now, this is saying, "Show an error if there are more than four modBoxes, but if there are five modBoxes, only show an error if the last one is filled in". I don't think that's exactly what we want. I think we want something simpler, like "Show an error if there are more than 4 filled-in modBoxes."

Notice that in the first one, we're talking about the "last" box. Do we really care if it's the _last_ box? Does the order really matter? What if they make five modBoxes but fill in the bottom four? Or the first two and the last two? It's easy to imagine scenarios where that happens, e.g. a user adds 5 moderators, the click Done, the error pops up that tells them they can only have four, so they remove the person they need the least from the middle of the list.

This is what I was referring to before when I was talking about a dependency on ordering. Our code right now depends on things being in a certain order even though we don't actually care if they're in a certain order. Even though the values in the modBoxes will almost always be in the order we depend on, sometimes they won't, and those users will experience a bug.

Try expressing that condition I gave above: "Show an error if there are more than 4 filled-in modBoxes."